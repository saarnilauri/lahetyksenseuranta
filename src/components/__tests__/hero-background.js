import React from "react";
import renderer from "react-test-renderer";

import HeroBackground from "../hero-background.js";

describe("HeroBackground", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<HeroBackground>Test</HeroBackground>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
