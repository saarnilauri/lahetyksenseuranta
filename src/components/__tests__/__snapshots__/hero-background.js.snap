// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeroBackground renders correctly 1`] = `
<div
  className="bg-gray-900 bg-center bg-cover bg-no-repeat hero-background relative bg-hero-lg border-b-2 border-white mb-10"
  style={
    Object {
      "backgroundImage": "url(test-file-stub)",
    }
  }
>
  Test
</div>
`;
